tsconfig.dev.json

```diff
@@ -0,0 +1,6 @@
+{
+  "extends": "./tsconfig.json",
+  "compilerOptions": {
+    "noEmit": true
```

Comment:
Please short comment on why does this should to be different for dev/prod

Reply:
this is just a convenience thing for me so that I can use VSCode to run `tsc --watch` to check the entire project for type errors while i'm coding rather than just the current file, but it's not essential—I can just do it for myself. would that be preferable?

Reply:
Sounds like it would be useful for others! Can you add the above to this file (assuming it allows for JSON comments?)

Reply:
yep!